src/index.html

```diff
@@ -10,23 +10,47 @@
       http-equiv="X-UA-Compatible"
       content="ie=edge"
     />
-    <title>Document</title>
+    <title>Search</title>
     <link
       rel="stylesheet"
       href="style.css"
     />
   </head>
   <body>
-    <input
-      type="text"
-      data-qa="keypress"
-      placeholder="Try “Los Angeles“"
-    />
+    <form
+      action="#"
+      data-qa="big"
+      class="search search--big"
+    >
+      <label
+        for="search__input--big"
+        class="search__label search__label--big"
+      >
+        <input
+          class="search__input search__input--big"
+          type="text"
+          data-qa="keypress"
+          placeholder="Try “Los Angeles“"
+        />
+      </label>
+    </form>
 
-    <input
-      type="text"
-      data-qa="hover"
-      placeholder="Try “Los Angeles“"
-    />
+    <form
+      action="#"
+      data-qa="small"
+      class="search search--small"
+    >
+      <label
+        for="search__input--small"
+        class="search__label search__label--small"
+      >
```

Comment:
The `for` attribute of the `label` element should be the same as the `id` of the `input` element to bind them together. However, your `input` element doesn't have an `id` attribute. Please, add `id` attribute to your `input` elements and make sure it matches the `for` attribute of the corresponding `label`.